pkg/exchange/okex/exchange.go

```diff
@@ -55,6 +55,7 @@ const (
 	defaultQueryLimit = 100
 
 	maxHistoricalDataQueryPeriod = 90 * 24 * time.Hour
+	threeDaysHistoricalPeriod    = 3 * 24 * time.Hour
```

Comment:
use maxTradeHistoryPeriod ?

Reply:
We already have one that supports 90 days, called `maxHistoricalDataQueryPeriod`. 

And now we have another one that's only 3 days, so it's called `threeDaysHistoricalPeriod` XD.